### DISCUSSION TITLE
The customer journey (map)

### DISCUSSION THREAD

**Discussion by MegTei** [filecoin-project/notary-governance#232] [Ideas]
Human centred governance journey map: 
**The pitch (why do this)**
1 Our work and processes would benefit from designing the full stack customer journey across all actors. There are goals for Filecoin, and some from the customer perspective and it would benefit from the full picture with RKH and Notaries (motivations, friction, opportunities etc)
2 Has the potential to vastly improve Client application + onboarding experience: co-create with clients to design the experience they need.
3 The Map extends to the ecosystem, how do all the awesome dApps and solutions the community are building fit and and how can they be used

**Next steps**
This task will come out of the Market assessment, it can be split out to accelerate if needed in 2021

**Comment by MegTei** [comment]
Further info from todays governance call. Seeking Notary input/ feedback
[232V0.1.pdf](https://github.com/filecoin-project/notary-governance/files/7248178/232V0.1.pdf)
Related to https://github.com/filecoin-project/notary-governance/discussions/229#discussion-3573734

**Reply by andrewxhill** [reply]
great work! looks like some useful thinking here
